db/schema.rb

```diff
@@ -225,12 +225,24 @@
     t.index ["source_id"], name: "index_service_plans_on_source_id"
   end
 
+  create_table "source_types", force: :cascade do |t|
+    t.string "name"
+    t.string "display_name"
+    t.string "vendor"
+    t.jsonb "schema"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["name"], name: "index_source_types_on_name", unique: true
+  end
+
   create_table "sources", force: :cascade do |t|
     t.string "name"
     t.string "uid"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
     t.bigint "tenant_id", null: false
+    t.bigint "source_type_id", null: false
```

Comment:
If this is null false, you might need a data migration to fix existing records.

Reply:
Removed the non-null constraint until we figure out the seeding since we wouldn't have anything to set it to in a data migration